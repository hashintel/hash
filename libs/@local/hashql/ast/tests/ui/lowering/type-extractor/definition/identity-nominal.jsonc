//@ run: pass
//@ description: Tests that newtypes get nominal identity with qualified path in the type system
["newtype", "Distinct", "Number", "_"]
