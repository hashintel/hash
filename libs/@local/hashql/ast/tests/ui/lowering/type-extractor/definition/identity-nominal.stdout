Expr#4294967040@13
  ExprKind (Underscore)

------------------------

type Distinct:0 = Distinct:0[Number]